model/host/host.go

```diff
@@ -62,7 +62,7 @@ type Host struct {
 
 	// the task that is currently running on the host
 	RunningTask             string `bson:"running_task,omitempty" json:"running_task,omitempty"`
-	RunningTaskExecution    int    `bson:"running_task_execution,omitempty" json:"running_task_execution,omitempty"`
+	RunningTaskExecution    int    `bson:"running_task_execution" json:"running_task_execution"`
```

Comment:
Did you find instances where the field didn't exist? If there are cases of that, we will still have to case against that since someone may be trying to restart and old task.